### ISSUE TITLE
amico.core.setup() error

### ISSUE THREAD

**Issue by kingLeicester** [daducci/AMICO#71]
I was having trouble installing spams, but resolved it via the solution provided for issue #68.

However, when I do:

amico.core.setup()

I get this error:

AttributeError: 'module' object has no attribute 'core'

How can I resolve this error?

Thank you!

**Comment by fullbat** [comment]
Hi @kingLeicester,
'core' error is usually caused by a bad installation of spams module.
I suggest using the 2.6 version (you can find here [here](http://spams-devel.gforge.inria.fr/downloads.html)), that I've recently tested.

Matteo

**Comment by kingLeicester** [comment]
Hi @fullbat 

Thanks for your suggestion. I did try installing version 2.6 spams (instead of 2.6.1), but still having the same error. Assuming this is what you suggested, I might be doing something incorrectly. Would you be able to tell me little more about how you tested it?

Thanks!

**Comment by fullbat** [comment]
@kingLeicester did you install the DIPY module inside your python environment? In other words, can you import dipy with python 2?

**Comment by kingLeicester** [comment]
@fullbat Yes I have. I can import dipy. Does AMICO module only work with specific python version? I am currently using 2.7.15. I have re-installed conda, dipy, spams 2.6, and AMICO  from scratch as well, but no luck there either. Any other thoughts would be helpful!

**Comment by fullbat** [comment]
@kingLeicester are you trying to import amico inside the AMICO installation folder? In that case, open another terminal in a different folder and try to import it.
From what I know it shouldn't need a specific python version.

**Comment by kingLeicester** [comment]
@fullbat. Well, I don't have an AMICO installation. I use conda and I did "pip install amico". If I should use a psychical setup.py install process, let me know!

**Comment by daducci** [comment]
Hi @kingLeicester ,
are you sure that, by issuing `pip install amico`, you are not trying to install this `amico` library? https://pypi.org/project/amico/

**Comment by kingLeicester** [comment]
hi @daducci 
I think that was the problem. I downloaded the actual setup.py and did pip install. I finally got it to work. Thanks for your help @fullbat and @daducci
